binder/start

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+export BOKEH_ALLOW_WS_ORIGIN=hub.gke2.mybinder.org
+echo '' >> panel/template/base.py
+echo 'BasicTemplate.param.site_url.default="./"' >> panel/template/base.py
+export DISPLAY=:99.0
+export PYVISTA_OFF_SCREEN=true
+export PYVISTA_USE_IPYVTK=true
+export PYVISTA_PLOT_THEME=document
+which Xvfb
+Xvfb :99 -screen 0 1280x1024x24 > /dev/null 2>&1 &
+sleep 3
+jupyter trust examples/**/*.ipynb
+jupyter trust examples/**/**/*.ipynb
```

Comment:
I'm not a bash expert, but doesn't `**` indicate recursive directory searching nowadays? If so, isn't the second trust call redundant?

Reply:
I thought so. But it did not work like that for me. So I added it twice. That works.